django_project/lesson/templates/worksheet/print.html

```diff
@@ -196,6 +196,14 @@ <h2>Further reading:</h2>
     </ul>
 </section>
 <footer>
+     <div class="col-lg-12" style="margin-bottom: 10px;
```

Comment:
There is no bootstrap in that page. Please remove it. Look a few lines above for the further reading.